flax/core/lift.py

```diff
@@ -232,6 +232,7 @@ def repack(inner_scope_tree):
             # Some lifted transforms like scan return redundant variables.
             continue
           for var_name, value in collection.items():
+            print('pack', col_name, var_name, value)
```

Comment:
Remove?

Reply:
lol.  thanks for noticing this, I was going crazy wondering what that doctest failure was about - I think this is an old edit that vscode held onto and was accidentally saved into this PR.